kapitan/cli.py

```diff
@@ -357,6 +359,10 @@ def _search_imports(cwd, imp):
 
     elif cmd == 'refs':
         ref_controller = RefController(args.refs_path)
+        if args.target_name:
+            ref_controller.target = args.target_name
```

Comment:
Can you make another Constructor for RefController and pass these extra arguments with that one instead?
Defining class variables from an external class can easily lead to harder to trace bugs in the future. 
Also name those variables `target_name` and `inventory_path` so that the name tells exactly what they are. :)